packages/taco/src/conditions/schemas/jwt.ts

```diff
@@ -0,0 +1,20 @@
+import { z } from 'zod';
+
+import { baseConditionSchema } from './common';
+import { contextParamSchema } from './context';
+
+export const JWT_PARAM_DEFAULT = ':jwtToken';
+
+export const JWTConditionType = 'jwt';
+
+export const jwtConditionSchema = baseConditionSchema.extend({
+  conditionType: z.literal(JWTConditionType).default(JWTConditionType),
+  publicKey: z.string().optional(),
+  expectedIssuer: z.string().optional(),
+  subject: contextParamSchema.optional(),
+  expirationWindow: z.number().int().nonnegative().optional(),
+  issuedWindow: z.number().int().nonnegative().optional(),
```

Comment:
@vzotova let's comment/drop these lines for the moment. The corresponding PR on the nucypher side will only validate the expected issuer and public key. We can easily add more restrictions like these once we have more clarity on what's actually needed.

Reply:
✔️